docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 services:
   traefik:
     profiles: ["studio"]
-    image: traefik:v3.2
+    image: traefik:v3.3.3
+    pull_policy: always
```

Comment:
I don't think it's necessary to mention pull_policy always. When the image changes, docker should be intelligent enough to pull it again
```suggestion
```